chalice/app.py

```diff
@@ -107,18 +109,14 @@ def wrapper(*args, **kwargs):
 
 def get_chalice_app(flask_app) -> DSSChaliceApp:
     app = DSSChaliceApp(app_name=flask_app.name, configure_logs=False)
+    ex = ThreadPoolExecutor(max_workers=2)
```

Comment:
I think this is invoked for every request so I would put this at module (global) scope.